MH4U Dex/ItemMonsterSourcesTableViewController.m

```diff
@@ -74,9 +77,8 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
     UIStoryboard *storyBoard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-    MonsterDetailsViewController *monsterVC = [storyBoard instantiateViewControllerWithIdentifier:@"MonsterDetailsViewController"];
-    monsterVC.managedObjectContext = self.managedObjectContext;
-    monsterVC.monsterName = [self dropAtIndexPath:indexPath].monsterSource.name;
+    MonsterContainerViewController *monsterVC = [storyBoard instantiateViewControllerWithIdentifier:@"MonsterDetailsViewController"];
```

Comment:
Would it make sense to replace the string literal here with something like `NSStringFromClass([MonsterDetailsViewController class])`?